### PR TITLE
Get the id of an user when login

### PR DESCRIPTION
**Pull request by M0UTOO**

### CODE REVIEW

File: users/users.c [hunk 1]
```diff
@@ -118,6 +150,8 @@ void loginUser(sqlite3 *db, char *zErrMsg, int rc, char *username, char *passwor
             *connected = 2;
         }else
         {
+        
+        *id = getUserID(db, zErrMsg, rc, username, password);
         printf("You are now connected !\n");
         printf("Welcome %s ^^\n\n", username);
```
Comment (Mil0w0) [inline users/users.c:156]: remove ^^ it is awful

**Review by Mil0w0 — CHANGES_REQUESTED** [review]
almost perfekt

File: users/users.c (hunk 1 above)
Comment (M0UTOO) [reply users/users.c:156]: Good luck trying to convince me ^^

**Review by Mil0w0 — APPROVED** [review]
OK

**Pull request merged**